### PR TITLE
Aiopeewee, aiomysql

### PR DESCRIPTION
**Pull request by kszucs**

### CODE REVIEW

**Comment by kszucs** [comment]
@conda-forge/staged-recipes please review

File: recipes/aiopeewee/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    - aiomysql >=0.0.9
+
+test:
+  imports:
+    - aiopeewee
+
+about:
+  home: https://github.com/kszucs/aiopeewee
+  summary: "Asyncio interface for Peewee ORM"
+  license: MIT
```
Comment (dougalsutherland) [inline recipes/aiopeewee/meta.yaml]: I see this is your project: you should make a `LICENSE` file, put it in `MANIFEST.in`, and either make a new release with it or package it manually here until you do. (The MIT license requires redistributions to include a copy of the license text.)

File: recipes/aiopeewee/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [py<36]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pytest-runner
```
Comment (dougalsutherland) [inline recipes/aiopeewee/meta.yaml]: I see this is in the `setup_requires`, but it doesn't look like it's actually needed?

File: recipes/aiopeewee/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
```
Comment (dougalsutherland) [inline recipes/aiopeewee/meta.yaml]: Nit: drop this blank line.

File: recipes/aiomysql/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [py<33]
+
+requirements:
+  build:
+    - python
+    - setuptools
+
```
Comment (dougalsutherland) [inline recipes/aiomysql/meta.yaml]: Nit: drop this blank line.

File: recipes/aiomysql/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+about:
+  home: https://github.com/aio-libs/aiomysql
+  summary: "aiomysql is a library for accessing a MySQL database from the asyncio"
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - jettify
```
Comment (dougalsutherland) [inline recipes/aiomysql/meta.yaml]: @jettify, are you interested in [being a co-maintainer](http://conda-forge.readthedocs.io/en/latest/recipe.html#maintainer-role) for the conda-forge package of `aiomysql`?

Comment (jettify) [reply recipes/aiomysql/meta.yaml]: Thanks for your offer, unfortunately I am have lots of projects to maintain at https://github.com/aio-libs and doubt I will manage also contribute here.

File: recipes/aiopeewee/meta.yaml (hunk 2 above)
Comment (kszucs) [reply recipes/aiopeewee/meta.yaml]: The build has failed [previously](https://circleci.com/gh/kszucs/staged-recipes/38?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link)

File: recipes/aiopeewee/meta.yaml (hunk 1 above)
Comment (kszucs) [reply recipes/aiopeewee/meta.yaml]: Done

File: recipes/aiopeewee/meta.yaml (hunk 2 above)
Comment (dougalsutherland) [reply recipes/aiopeewee/meta.yaml]: Ah right well, if it's in `setup_requires` of course it'll try to download if not. Oops. :)

**Comment by kszucs** [comment]
@dougalsutherland is it good to merge?

File: recipes/aiopeewee/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [py<36]
```
Comment (dougalsutherland) [inline recipes/aiopeewee/meta.yaml]: Does it not work for python 3.5? This isn't documented in the package as far as I can tell.

Comment (kszucs) [reply recipes/aiopeewee/meta.yaml]: No, aiopeewee uses async generators too which requires 3.6

**Comment by kszucs** [comment]
@dougalsutherland any todos left?

**Comment by dougalsutherland** [comment]
Oops, didn't check this before but apparently [`pymysql`](https://github.com/PyMySQL/PyMySQL) isn't in conda-forge yet, and ideally we package all the dependencies.

I'm throwing together a quick PR for it (it's pure python and easy); do you want to be a maintainer?

**Comment by dougalsutherland** [comment]
Waiting for pymysql to build and then I'll merge this.

**Comment by kszucs** [comment]
Sorry, I forgot pymysql. Thank You!

**Comment by kszucs** [comment]
BTW should I prefer noarch for pure python packages?

**Comment by dougalsutherland** [comment]
Yep, but only if it also works on all python versions and OSes (unlike these ones). It lightens the load on the build servers considerably when it works.

**Comment by dougalsutherland** [comment]
pymysql built, so merging now. Thanks @kszucs!

**Pull request merged**